docs/using-hive.md

```diff
@@ -104,15 +104,45 @@ spec:
     name: mycluster-ssh-key
 ```
 
-The hiveutil CLI (see `make hiveutil`) offers a create-cluster command for generating a cluster deployment and submitting it to the cluster defined by your current kubeconfig.
+The hiveutil CLI (see `make hiveutil`) offers a create-cluster command for generating a cluster deployment and submitting it to the Hive cluster using your current kubeconfig.
+
+To view what create-cluster generates, *without* submitting it to the API server, add `-o yaml` to the above command. If you need to make any changes not supported by create-cluster options, the output can be saved, edited, and then submitted with `oc apply`.
+
+By default this command assumes the latest Hive master CI build, and the latest OpenShift stable release. `--release-image` can be specified to control which OpenShift release image to install in the cluster.
+
+#### Create Cluster on AWS
+
+Credentials will be read from your AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY environment variables. Alternatively you can specify an AWS credentials file with --creds-file.
 
 ```bash
-bin/hiveutil create-cluster --base-domain=mydomain.example.com mycluster
+bin/hiveutil create-cluster --base-domain=mydomain.example.com --cloud=aws mycluster
```

Comment:
Are you including `--cloud=aws` because we want to encourage explicitly setting the cloud to AWS instead of relying on AWS being the default?

Reply:
I felt it was more clear in this context in the face of the two commands below to show each explicitly setting their cloud.

Reply:
Sounds good.